tests/unit/lms/services/blackboard_api/_schemas_test.py

```diff
@@ -107,16 +108,46 @@ def test_it(self, group_sets_response):
 
         result = schema.parse()
 
-        assert result == [{"id": "GROUP_1", "name": "GROUP 1"}]
+        assert result == [{"id": "GROUP_SET_1", "name": "GROUP SET 1"}]
 
     @pytest.fixture
     def group_sets_response(self):
+        return {
+            "results": [
+                {
+                    "id": "GROUP_SET_1",
+                    "name": "GROUP SET 1",
+                }
+            ]
+        }
+
+
+class TestBlackboardListGroupSetGroups:
+    def test_it(self, group_set_groups_response):
+        schema = BlackboardListGroupSetGroups(
+            factories.requests.Response(json_data=group_set_groups_response)
+        )
+
+        result = schema.parse()
+
+        assert result == [
+            {"id": "GROUP_1", "name": "GROUP 1", "groupSetId": "GROUP SET 1"},
+            {"id": "GROUP_2", "name": "GROUP 2"},
+        ]
```

Comment:
I think we probably want at least a `test_valid()` (with one or more valid test cases) and a `test_invalid()` (parametrized test cases for some invalid results that should raise `ValidationError`)